Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "Revive", "type": "bool" }
+      ]
+    },
+    "ObjPunchingBallSpawner": {
+      "fields": [
+        { "name": "Length", "type": "float32" }
+      ]
+    },
+    "ObjSlotSpawner": {
+      "fields": [
+        { "name": "Probabilities[0]", "type": "float32" },
+        { "name": "Probabilities[1]", "type": "float32" },
+        { "name": "Probabilities[2]", "type": "float32" },
+        { "name": "Probabilities[3]", "type": "float32" },
+        { "name": "Probabilities[4]", "type": "float32" }
+      ]
+    },
+    "ObjSprialDoorSpawner": {
+      "fields": [
+        { "name": "KeepVelocityDistance", "type": "float32" },
+        { "name": "Speed", "type": "float32" }
+      ]
+    },
+    "ObjWaterCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" }
+      ]
+    },
+    "ObjPhantomObjBreakableSpawner": {
+      "fields": [
+      ]
+    },
+    "ObjBombSpawner": {
+      "fields": [
+        { "name": "Counter", "type": "uint32" },
+        { "name": "SearchRadius", "type": "float32" }
+      ]
+    },
+    "ObjConnectionPipeSpawner": {
+      "fields": [
+        { "name": "reverse", "type": "bool" }
+      ]
+    },
+    "ObjEventLaserCancelCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" }
+      ]
+    },
+    "ObjPrismSpawner": {
+      "fields": [
+        { "name": "Pitch", "type": "float32" },
+        { "name": "Yaw", "type": "float32" },
+        { "name": "Target", "type": "vector3" },
+        { "name": "ObjectID", "type": "object_reference" },
+        { "name": "Type", "type": "ObjPrismSpawner::Type" },
+        { "name": "CameraID", "type": "object_reference" }
+      ]
+    },
+    "ObjFiberSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionDepth", "type": "float32" },
+        { "name": "PathID", "type": "uint32" },
+        { "name": "TargetCamera", "type": "object_reference" },
+        { "name": "EventType", "type": "ObjFiberSpawner::EventType" },
+        { "name": "IsExitOutside", "type": "bool" },
+        { "name": "IsExit2D", "type": "bool" },
+        { "name": "ExitType", "type": "ObjFiberSpawner::ExitType" },
+        { "name": "LaserSpeed", "type": "float32" },
+        { "name": "IsChangeSpeed", "type": "bool" }
+      ]
+    },
+    "ObjCogWheelSpawner": {
+      "fields": [
+        { "name": "num", "type": "uint32" }
+      ]
+    },
+    "ObjRotationGlobeSpawner": {
+      "fields": [
+        { "name": "SizeType", "type": "ObjRotationGlobeSpawner::SizeType" },
+        { "name": "IsReverseStart", "type": "bool" }
+      ]
+    },
```

Comment:
Unused object. Parameter names are based on `PlaGearFloor` from Sonic Generations and in-game functionality. Set data  parser function (0x801CD90C, NTSC-J Wii) was used for additional help.

Further information on the object at:
https://hedgedocs.com/discoveries/sonic2010/objects/unused/#rotationglobe